src/server/getters/get-reporting-fees.ts

```diff
@@ -265,7 +271,11 @@ function getStakedRepResults(db: Knex, reporter: Address, universe: Address, cal
     initialReporters: (next: AsyncCallback) => initialReportersQuery.asCallback(next),
   }, (err: Error|null, result: StakedRepResults) => {
     if (err) return callback(err);
+
+    const marketDisputed: MarketsDisputedMap = {};
+
     let fees = _.reduce(result.crowdsourcers, (acc: RepStakeResults, feeWindowCompletionStake: FeeWindowCompletionStakeRow) => {
+      marketDisputed[feeWindowCompletionStake.marketId] = true;
```

Comment:
Aren't we only returning crowdsourcers for which the user has a stake?

Reply:
this is checking that the market was disputed, not that the user has stake.

Reply:
Forgot we left-joined! 👍